ports/psoc6/Makefile

```diff
@@ -13,15 +13,35 @@ ifeq ($(wildcard $(BOARD_DIR)/.),)
   $(error Invalid BOARD specified)
 endif
 
+# Supporting variables for make-pins.py script execution
+MAKE_PINS_CSV = ./boards/make-pins-csv.py
+MAKE_PINS = ./boards/make-pins.py
+
+ifeq ($(BOARD), CY8CPROTO-062-4343W)
```

Comment:
These can go in the corresponding mpconfigport.mk or?

Reply:
Or rather more suitable in mpconfigboard.mk? Because each board will have one such variable set no?

Reply:
Yes, sorry. I meant mpconfigboard.mk sorry :)

Reply:
Agreed